File: iCyption2022/CftTest/2D/solution.py
import math

def pythagoras(a, b):
    return math.sqrt(a*a + b*b)

n = int(input())

X, Y = 0, 0

for i in range(n) :
    dist, deg = map(float, input().split())

    X += dist * math.cos(math.radians(deg))
    Y += dist * math.sin(math.radians(deg))

print("{:.4f}".format(pythagoras(X, Y)), end=" ")
if (X == 0) :
    print("{:.4f}".format(90))
else:
    print("{:.4f}".format(math.degrees(math.atan(Y/X))))

File: iCyption2022/CftTest/2D/solution.java
import java.util.*;
import java.lang.Math.*;

public class solution {

    static Scanner cin = new Scanner(System.in);

    public static double pythagoras(double a, double b) {
        return Math.sqrt(a*a + b*b);
    }

    public static void main(String[] args) {
        long n = cin.nextLong();

        double X = 0, Y = 0;

        for (long i = 0; i < n; i++) {
            double dist = cin.nextDouble(), deg = cin.nextDouble();

            X += dist * Math.cos(Math.toRadians(deg));
            Y += dist * Math.sin(Math.toRadians(deg));
        }

        System.out.println(String.format("%.4f %.4f", pythagoras(X, Y), Math.toDegrees(Math.atan(Y/X)) ));
    }
}